src/components/ItemValueLookup.vue

```diff
@@ -5,6 +5,7 @@
 		:error="error"
 		:searchForMenuItems="searchForItems"
 		:label="$i18n('query-builder-input-value-label')"
+		:tooltip="$i18n('query-builder-item-value-tooltip')"
```

Comment:
This is the i18n message that doesn't exist. :)

But that being said, it probably shouldn't be created, but we should inject that message in the ValueInput component. Because, as I understand it, we do not have different messages for the different datatypes, right? What do you think?

Reply:
oh yeah, I initially thought only item value needs it but during the change, I forgot to fix this one.